Binary files ../../../android-release-2.8/.DS_Store and ./.DS_Store differ
Binary files ../../../android-release-2.8/bin/Subsonic.apk and ./bin/Subsonic.apk differ
Binary files ../../../android-release-2.8/bin/classes.dex and ./bin/classes.dex differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/R$anim.class and ./bin/net/sourceforge/subsonic/androidapp/R$anim.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/R$array.class and ./bin/net/sourceforge/subsonic/androidapp/R$array.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/R$attr.class and ./bin/net/sourceforge/subsonic/androidapp/R$attr.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/R$color.class and ./bin/net/sourceforge/subsonic/androidapp/R$color.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/R$drawable.class and ./bin/net/sourceforge/subsonic/androidapp/R$drawable.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/R$id.class and ./bin/net/sourceforge/subsonic/androidapp/R$id.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/R$layout.class and ./bin/net/sourceforge/subsonic/androidapp/R$layout.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/R$menu.class and ./bin/net/sourceforge/subsonic/androidapp/R$menu.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/R$plurals.class and ./bin/net/sourceforge/subsonic/androidapp/R$plurals.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/R$string.class and ./bin/net/sourceforge/subsonic/androidapp/R$string.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/R$style.class and ./bin/net/sourceforge/subsonic/androidapp/R$style.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/R$xml.class and ./bin/net/sourceforge/subsonic/androidapp/R$xml.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/R.class and ./bin/net/sourceforge/subsonic/androidapp/R.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$1.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$10.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$10.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$11.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$11.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$12$1.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$12$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$12.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$12.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$13.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$13.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$14.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$14.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$15.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$15.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$2.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$3.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$3.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$4.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$4.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$5.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$5.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$6.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$6.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$7.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$7.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$8.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$8.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$9.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$9.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$SongListAdapter.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity$SongListAdapter.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity.class and ./bin/net/sourceforge/subsonic/androidapp/activity/DownloadActivity.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/ErrorActivity.class and ./bin/net/sourceforge/subsonic/androidapp/activity/ErrorActivity.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/HelpActivity$1.class and ./bin/net/sourceforge/subsonic/androidapp/activity/HelpActivity$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/HelpActivity$2.class and ./bin/net/sourceforge/subsonic/androidapp/activity/HelpActivity$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/HelpActivity$HelpClient.class and ./bin/net/sourceforge/subsonic/androidapp/activity/HelpActivity$HelpClient.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/HelpActivity.class and ./bin/net/sourceforge/subsonic/androidapp/activity/HelpActivity.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/LyricsActivity$1.class and ./bin/net/sourceforge/subsonic/androidapp/activity/LyricsActivity$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/LyricsActivity.class and ./bin/net/sourceforge/subsonic/androidapp/activity/LyricsActivity.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/MainActivity$1.class and ./bin/net/sourceforge/subsonic/androidapp/activity/MainActivity$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/MainActivity$2.class and ./bin/net/sourceforge/subsonic/androidapp/activity/MainActivity$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/MainActivity$3.class and ./bin/net/sourceforge/subsonic/androidapp/activity/MainActivity$3.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/MainActivity.class and ./bin/net/sourceforge/subsonic/androidapp/activity/MainActivity.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/PlayVideoActivity$Client.class and ./bin/net/sourceforge/subsonic/androidapp/activity/PlayVideoActivity$Client.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/PlayVideoActivity.class and ./bin/net/sourceforge/subsonic/androidapp/activity/PlayVideoActivity.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/QueryReceiverActivity.class and ./bin/net/sourceforge/subsonic/androidapp/activity/QueryReceiverActivity.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SearchActivity$1.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SearchActivity$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SearchActivity$2.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SearchActivity$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SearchActivity$3.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SearchActivity$3.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SearchActivity.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SearchActivity.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$1.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$10.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$10.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$11$1.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$11$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$11.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$11.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$12.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$12.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$13.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$13.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$14.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$14.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$2.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$3.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$3.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$4.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$4.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$5.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$5.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$6.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$6.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$7.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$7.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$8.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$8.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$9.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$9.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$LoadTask.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity$LoadTask.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectArtistActivity$1.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectArtistActivity$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectArtistActivity$2.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectArtistActivity$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectArtistActivity$3.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectArtistActivity$3.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectArtistActivity.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectArtistActivity.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectPlaylistActivity$1.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectPlaylistActivity$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectPlaylistActivity$2.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectPlaylistActivity$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectPlaylistActivity$PlaylistAdapter.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectPlaylistActivity$PlaylistAdapter.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SelectPlaylistActivity.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SelectPlaylistActivity.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity$1.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity$2.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity$3.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity$3.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity$4.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity$4.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity$5.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity$5.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity$ServerSettings$1.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity$ServerSettings$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity$ServerSettings$2.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity$ServerSettings$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity$ServerSettings.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity$ServerSettings.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SettingsActivity.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity$1.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity$2.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity$3.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity$3.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity$4.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity$4.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity$5.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity$5.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity$6.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity$6.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity.class and ./bin/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/activity/VoiceQueryReceiverActivity.class and ./bin/net/sourceforge/subsonic/androidapp/activity/VoiceQueryReceiverActivity.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/domain/Artist.class and ./bin/net/sourceforge/subsonic/androidapp/domain/Artist.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/domain/Indexes.class and ./bin/net/sourceforge/subsonic/androidapp/domain/Indexes.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/domain/Lyrics.class and ./bin/net/sourceforge/subsonic/androidapp/domain/Lyrics.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/domain/MusicDirectory$Entry.class and ./bin/net/sourceforge/subsonic/androidapp/domain/MusicDirectory$Entry.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/domain/MusicDirectory.class and ./bin/net/sourceforge/subsonic/androidapp/domain/MusicDirectory.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/domain/MusicFolder.class and ./bin/net/sourceforge/subsonic/androidapp/domain/MusicFolder.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/domain/PlayerState.class and ./bin/net/sourceforge/subsonic/androidapp/domain/PlayerState.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/domain/Playlist.class and ./bin/net/sourceforge/subsonic/androidapp/domain/Playlist.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/domain/RepeatMode$1.class and ./bin/net/sourceforge/subsonic/androidapp/domain/RepeatMode$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/domain/RepeatMode$2.class and ./bin/net/sourceforge/subsonic/androidapp/domain/RepeatMode$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/domain/RepeatMode$3.class and ./bin/net/sourceforge/subsonic/androidapp/domain/RepeatMode$3.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/domain/RepeatMode.class and ./bin/net/sourceforge/subsonic/androidapp/domain/RepeatMode.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/domain/SearchCritera.class and ./bin/net/sourceforge/subsonic/androidapp/domain/SearchCritera.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/domain/SearchResult.class and ./bin/net/sourceforge/subsonic/androidapp/domain/SearchResult.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/domain/ServerInfo.class and ./bin/net/sourceforge/subsonic/androidapp/domain/ServerInfo.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/domain/Version.class and ./bin/net/sourceforge/subsonic/androidapp/domain/Version.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/provider/SearchSuggestionProvider.class and ./bin/net/sourceforge/subsonic/androidapp/provider/SearchSuggestionProvider.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/provider/SubsonicAppWidgetProvider.class and ./bin/net/sourceforge/subsonic/androidapp/provider/SubsonicAppWidgetProvider.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/receiver/BluetoothIntentReceiver.class and ./bin/net/sourceforge/subsonic/androidapp/receiver/BluetoothIntentReceiver.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/receiver/MediaButtonIntentReceiver.class and ./bin/net/sourceforge/subsonic/androidapp/receiver/MediaButtonIntentReceiver.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/CachedMusicService.class and ./bin/net/sourceforge/subsonic/androidapp/service/CachedMusicService.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/DownloadFile$DownloadTask$1.class and ./bin/net/sourceforge/subsonic/androidapp/service/DownloadFile$DownloadTask$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/DownloadFile$DownloadTask.class and ./bin/net/sourceforge/subsonic/androidapp/service/DownloadFile$DownloadTask.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/DownloadFile.class and ./bin/net/sourceforge/subsonic/androidapp/service/DownloadFile.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/DownloadService.class and ./bin/net/sourceforge/subsonic/androidapp/service/DownloadService.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceImpl$1.class and ./bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceImpl$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceImpl$2.class and ./bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceImpl$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceImpl$BufferTask.class and ./bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceImpl$BufferTask.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceImpl.class and ./bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceImpl.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceLifecycleSupport$1.class and ./bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceLifecycleSupport$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceLifecycleSupport$2.class and ./bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceLifecycleSupport$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceLifecycleSupport$3.class and ./bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceLifecycleSupport$3.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceLifecycleSupport$MyPhoneStateListener.class and ./bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceLifecycleSupport$MyPhoneStateListener.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceLifecycleSupport$State.class and ./bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceLifecycleSupport$State.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceLifecycleSupport.class and ./bin/net/sourceforge/subsonic/androidapp/service/DownloadServiceLifecycleSupport.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/MediaStoreService.class and ./bin/net/sourceforge/subsonic/androidapp/service/MediaStoreService.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/MusicService.class and ./bin/net/sourceforge/subsonic/androidapp/service/MusicService.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/MusicServiceFactory.class and ./bin/net/sourceforge/subsonic/androidapp/service/MusicServiceFactory.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/OfflineMusicService.class and ./bin/net/sourceforge/subsonic/androidapp/service/OfflineMusicService.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/RESTMusicService$1.class and ./bin/net/sourceforge/subsonic/androidapp/service/RESTMusicService$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/RESTMusicService.class and ./bin/net/sourceforge/subsonic/androidapp/service/RESTMusicService.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/Scrobbler$1.class and ./bin/net/sourceforge/subsonic/androidapp/service/Scrobbler$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/Scrobbler.class and ./bin/net/sourceforge/subsonic/androidapp/service/Scrobbler.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/parser/AbstractParser.class and ./bin/net/sourceforge/subsonic/androidapp/service/parser/AbstractParser.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/parser/AlbumListParser.class and ./bin/net/sourceforge/subsonic/androidapp/service/parser/AlbumListParser.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/parser/ErrorParser.class and ./bin/net/sourceforge/subsonic/androidapp/service/parser/ErrorParser.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/parser/IndexesParser.class and ./bin/net/sourceforge/subsonic/androidapp/service/parser/IndexesParser.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/parser/LicenseParser.class and ./bin/net/sourceforge/subsonic/androidapp/service/parser/LicenseParser.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/parser/LyricsParser.class and ./bin/net/sourceforge/subsonic/androidapp/service/parser/LyricsParser.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/parser/MusicDirectoryEntryParser.class and ./bin/net/sourceforge/subsonic/androidapp/service/parser/MusicDirectoryEntryParser.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/parser/MusicDirectoryParser.class and ./bin/net/sourceforge/subsonic/androidapp/service/parser/MusicDirectoryParser.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/parser/MusicFoldersParser.class and ./bin/net/sourceforge/subsonic/androidapp/service/parser/MusicFoldersParser.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/parser/PlaylistParser.class and ./bin/net/sourceforge/subsonic/androidapp/service/parser/PlaylistParser.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/parser/PlaylistsParser.class and ./bin/net/sourceforge/subsonic/androidapp/service/parser/PlaylistsParser.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/parser/RandomSongsParser.class and ./bin/net/sourceforge/subsonic/androidapp/service/parser/RandomSongsParser.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/parser/SearchResult2Parser.class and ./bin/net/sourceforge/subsonic/androidapp/service/parser/SearchResult2Parser.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/parser/SearchResultParser.class and ./bin/net/sourceforge/subsonic/androidapp/service/parser/SearchResultParser.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/parser/SubsonicRESTException.class and ./bin/net/sourceforge/subsonic/androidapp/service/parser/SubsonicRESTException.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/parser/VersionParser.class and ./bin/net/sourceforge/subsonic/androidapp/service/parser/VersionParser.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/ssl/SSLSocketFactory.class and ./bin/net/sourceforge/subsonic/androidapp/service/ssl/SSLSocketFactory.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/ssl/TrustManagerDecorator.class and ./bin/net/sourceforge/subsonic/androidapp/service/ssl/TrustManagerDecorator.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/ssl/TrustSelfSignedStrategy.class and ./bin/net/sourceforge/subsonic/androidapp/service/ssl/TrustSelfSignedStrategy.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/service/ssl/TrustStrategy.class and ./bin/net/sourceforge/subsonic/androidapp/service/ssl/TrustStrategy.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/AlbumView.class and ./bin/net/sourceforge/subsonic/androidapp/util/AlbumView.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/ArtistAdapter.class and ./bin/net/sourceforge/subsonic/androidapp/util/ArtistAdapter.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/BackgroundTask.class and ./bin/net/sourceforge/subsonic/androidapp/util/BackgroundTask.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/CacheCleaner$1.class and ./bin/net/sourceforge/subsonic/androidapp/util/CacheCleaner$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/CacheCleaner.class and ./bin/net/sourceforge/subsonic/androidapp/util/CacheCleaner.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/CancellableTask$1.class and ./bin/net/sourceforge/subsonic/androidapp/util/CancellableTask$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/CancellableTask$OnCancelListener.class and ./bin/net/sourceforge/subsonic/androidapp/util/CancellableTask$OnCancelListener.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/CancellableTask.class and ./bin/net/sourceforge/subsonic/androidapp/util/CancellableTask.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/Constants.class and ./bin/net/sourceforge/subsonic/androidapp/util/Constants.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/EntryAdapter.class and ./bin/net/sourceforge/subsonic/androidapp/util/EntryAdapter.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/ErrorDialog$1.class and ./bin/net/sourceforge/subsonic/androidapp/util/ErrorDialog$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/ErrorDialog$2.class and ./bin/net/sourceforge/subsonic/androidapp/util/ErrorDialog$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/ErrorDialog.class and ./bin/net/sourceforge/subsonic/androidapp/util/ErrorDialog.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/FileUtil.class and ./bin/net/sourceforge/subsonic/androidapp/util/FileUtil.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/HorizontalSlider$OnSliderChangeListener.class and ./bin/net/sourceforge/subsonic/androidapp/util/HorizontalSlider$OnSliderChangeListener.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/HorizontalSlider.class and ./bin/net/sourceforge/subsonic/androidapp/util/HorizontalSlider.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/ImageLoader$Task$1.class and ./bin/net/sourceforge/subsonic/androidapp/util/ImageLoader$Task$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/ImageLoader$Task.class and ./bin/net/sourceforge/subsonic/androidapp/util/ImageLoader$Task.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/ImageLoader.class and ./bin/net/sourceforge/subsonic/androidapp/util/ImageLoader.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/LRUCache$TimestampedValue.class and ./bin/net/sourceforge/subsonic/androidapp/util/LRUCache$TimestampedValue.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/LRUCache.class and ./bin/net/sourceforge/subsonic/androidapp/util/LRUCache.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/MergeAdapter$CascadeDataSetObserver.class and ./bin/net/sourceforge/subsonic/androidapp/util/MergeAdapter$CascadeDataSetObserver.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/MergeAdapter$EnabledSackAdapter.class and ./bin/net/sourceforge/subsonic/androidapp/util/MergeAdapter$EnabledSackAdapter.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/MergeAdapter.class and ./bin/net/sourceforge/subsonic/androidapp/util/MergeAdapter.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/ModalBackgroundTask$1.class and ./bin/net/sourceforge/subsonic/androidapp/util/ModalBackgroundTask$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/ModalBackgroundTask$2.class and ./bin/net/sourceforge/subsonic/androidapp/util/ModalBackgroundTask$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/ModalBackgroundTask$3$1.class and ./bin/net/sourceforge/subsonic/androidapp/util/ModalBackgroundTask$3$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/ModalBackgroundTask$3$2.class and ./bin/net/sourceforge/subsonic/androidapp/util/ModalBackgroundTask$3$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/ModalBackgroundTask$3.class and ./bin/net/sourceforge/subsonic/androidapp/util/ModalBackgroundTask$3.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/ModalBackgroundTask$4.class and ./bin/net/sourceforge/subsonic/androidapp/util/ModalBackgroundTask$4.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/ModalBackgroundTask.class and ./bin/net/sourceforge/subsonic/androidapp/util/ModalBackgroundTask.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/MyViewFlipper.class and ./bin/net/sourceforge/subsonic/androidapp/util/MyViewFlipper.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/Pair.class and ./bin/net/sourceforge/subsonic/androidapp/util/Pair.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/ProgressListener.class and ./bin/net/sourceforge/subsonic/androidapp/util/ProgressListener.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/SackOfViewsAdapter.class and ./bin/net/sourceforge/subsonic/androidapp/util/SackOfViewsAdapter.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/ShufflePlayBuffer$1.class and ./bin/net/sourceforge/subsonic/androidapp/util/ShufflePlayBuffer$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/ShufflePlayBuffer.class and ./bin/net/sourceforge/subsonic/androidapp/util/ShufflePlayBuffer.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/SilentBackgroundTask$1$1.class and ./bin/net/sourceforge/subsonic/androidapp/util/SilentBackgroundTask$1$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/SilentBackgroundTask$1$2.class and ./bin/net/sourceforge/subsonic/androidapp/util/SilentBackgroundTask$1$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/SilentBackgroundTask$1.class and ./bin/net/sourceforge/subsonic/androidapp/util/SilentBackgroundTask$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/SilentBackgroundTask.class and ./bin/net/sourceforge/subsonic/androidapp/util/SilentBackgroundTask.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/SimpleServiceBinder.class and ./bin/net/sourceforge/subsonic/androidapp/util/SimpleServiceBinder.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/SongView$1.class and ./bin/net/sourceforge/subsonic/androidapp/util/SongView$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/SongView.class and ./bin/net/sourceforge/subsonic/androidapp/util/SongView.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/TabActivityBackgroundTask$1$1.class and ./bin/net/sourceforge/subsonic/androidapp/util/TabActivityBackgroundTask$1$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/TabActivityBackgroundTask$1$2.class and ./bin/net/sourceforge/subsonic/androidapp/util/TabActivityBackgroundTask$1$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/TabActivityBackgroundTask$1.class and ./bin/net/sourceforge/subsonic/androidapp/util/TabActivityBackgroundTask$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/TabActivityBackgroundTask$2.class and ./bin/net/sourceforge/subsonic/androidapp/util/TabActivityBackgroundTask$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/TabActivityBackgroundTask.class and ./bin/net/sourceforge/subsonic/androidapp/util/TabActivityBackgroundTask.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/TimeLimitedCache.class and ./bin/net/sourceforge/subsonic/androidapp/util/TimeLimitedCache.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/Util$1.class and ./bin/net/sourceforge/subsonic/androidapp/util/Util$1.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/Util$2.class and ./bin/net/sourceforge/subsonic/androidapp/util/Util$2.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/Util$3.class and ./bin/net/sourceforge/subsonic/androidapp/util/Util$3.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/Util$4.class and ./bin/net/sourceforge/subsonic/androidapp/util/Util$4.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/Util$5.class and ./bin/net/sourceforge/subsonic/androidapp/util/Util$5.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/Util$6.class and ./bin/net/sourceforge/subsonic/androidapp/util/Util$6.class differ
Binary files ../../../android-release-2.8/bin/net/sourceforge/subsonic/androidapp/util/Util.class and ./bin/net/sourceforge/subsonic/androidapp/util/Util.class differ
Binary files ../../../android-release-2.8/bin/resources.ap_ and ./bin/resources.ap_ differ
diff -rupN ../../../android-release-2.8/gen/net/sourceforge/subsonic/androidapp/R.java ./gen/net/sourceforge/subsonic/androidapp/R.java
--- ../../../android-release-2.8/gen/net/sourceforge/subsonic/androidapp/R.java	1969-12-31 16:00:00.000000000 -0800
+++ ./gen/net/sourceforge/subsonic/androidapp/R.java	2011-06-17 08:34:07.000000000 -0700
@@ -0,0 +1,482 @@
+/* AUTO-GENERATED FILE.  DO NOT MODIFY.
+ *
+ * This class was automatically generated by the
+ * aapt tool from the resource data it found.  It
+ * should not be modified by hand.
+ */
+
+package net.sourceforge.subsonic.androidapp;
+
+public final class R {
+    public static final class anim {
+        public static final int push_down_in=0x7f040000;
+        public static final int push_down_out=0x7f040001;
+        public static final int push_up_in=0x7f040002;
+        public static final int push_up_out=0x7f040003;
+    }
+    public static final class array {
+        public static final int cacheSizeNames=0x7f060005;
+        public static final int cacheSizeValues=0x7f060004;
+        public static final int maxBitrateNames=0x7f060007;
+        public static final int maxBitrateValues=0x7f060006;
+        public static final int preloadCountNames=0x7f060003;
+        public static final int preloadCountValues=0x7f060002;
+        public static final int themeNames=0x7f060001;
+        public static final int themeValues=0x7f060000;
+    }
+    public static final class attr {
+    }
+    public static final class color {
+        public static final int appwidget_text=0x7f070003;
+        public static final int dividerColor=0x7f070002;
+        public static final int mediaControlBackground=0x7f070000;
+        public static final int mediaControlForeground=0x7f070001;
+        public static final int wheatBackground=0x7f070004;
+        public static final int wheatForeground1=0x7f070005;
+        public static final int wheatForeground2=0x7f070006;
+    }
+    public static final class drawable {
+        public static final int action_play_all=0x7f020000;
+        public static final int action_refresh=0x7f020001;
+        public static final int action_search=0x7f020002;
+        public static final int action_shuffle=0x7f020003;
+        public static final int actionbar_button=0x7f020004;
+        public static final int actionbar_button_normal=0x7f020005;
+        public static final int album_art_background=0x7f020006;
+        public static final int appwidget_bg=0x7f020007;
+        public static final int appwidget_button_center=0x7f020008;
+        public static final int appwidget_button_left=0x7f020009;
+        public static final int appwidget_button_right=0x7f02000a;
+        public static final int appwidget_divider=0x7f02000b;
+        public static final int appwidget_inner_focus_c=0x7f02000c;
+        public static final int appwidget_inner_focus_l=0x7f02000d;
+        public static final int appwidget_inner_focus_r=0x7f02000e;
+        public static final int appwidget_inner_press_c=0x7f02000f;
+        public static final int appwidget_inner_press_l=0x7f020010;
+        public static final int appwidget_inner_press_r=0x7f020011;
+        public static final int downloaded=0x7f020012;
+        public static final int downloading=0x7f020013;
+        public static final int ic_appwidget_music_next=0x7f020014;
+        public static final int ic_appwidget_music_pause=0x7f020015;
+        public static final int ic_appwidget_music_play=0x7f020016;
+        public static final int ic_menu_exit=0x7f020017;
+        public static final int ic_menu_help=0x7f020018;
+        public static final int ic_menu_remove_all=0x7f020019;
+        public static final int ic_menu_save=0x7f02001a;
+        public static final int ic_menu_screen_option=0x7f02001b;
+        public static final int ic_menu_settings=0x7f02001c;
+        public static final int ic_menu_shuffle=0x7f02001d;
+        public static final int icon=0x7f02001e;
+        public static final int launch=0x7f02001f;
+        public static final int list_item_artist=0x7f020020;
+        public static final int list_item_playlist=0x7f020021;
+        public static final int main_exit=0x7f020022;
+        public static final int main_header_icon=0x7f020023;
+        public static final int main_help=0x7f020024;
+        public static final int main_select_server=0x7f020025;
+        public static final int main_settings=0x7f020026;
+        public static final int main_shuffle=0x7f020027;
+        public static final int media_backward=0x7f020028;
+        public static final int media_backward_normal=0x7f020029;
+        public static final int media_backward_pressed=0x7f02002a;
+        public static final int media_forward=0x7f02002b;
+        public static final int media_forward_normal=0x7f02002c;
+        public static final int media_forward_pressed=0x7f02002d;
+        public static final int media_pause=0x7f02002e;
+        public static final int media_pause_normal=0x7f02002f;
+        public static final int media_pause_pressed=0x7f020030;
+        public static final int media_repeat_all=0x7f020031;
+        public static final int media_repeat_off=0x7f020032;
+        public static final int media_repeat_single=0x7f020033;
+        public static final int media_shuffle=0x7f020034;
+        public static final int media_shuffle_normal=0x7f020035;
+        public static final int media_shuffle_pressed=0x7f020036;
+        public static final int media_start=0x7f020037;
+        public static final int media_start_normal=0x7f020038;
+        public static final int media_start_pressed=0x7f020039;
+        public static final int media_stop=0x7f02003a;
+        public static final int media_stop_normal=0x7f02003b;
+        public static final int media_stop_pressed=0x7f02003c;
+        public static final int media_toggle_list=0x7f02003d;
+        public static final int media_toggle_list_normal=0x7f02003e;
+        public static final int media_toggle_list_pressed=0x7f02003f;
+        public static final int menu_browse=0x7f020040;
+        public static final int menu_home=0x7f020041;
+        public static final int menu_now_playing=0x7f020042;
+        public static final int menu_playlists=0x7f020043;
+        public static final int menu_search=0x7f020044;
+        public static final int menubar_button=0x7f020045;
+        public static final int menubar_button_disabled=0x7f020046;
+        public static final int menubar_button_normal=0x7f020047;
+        public static final int menubar_button_pressed=0x7f020048;
+        public static final int refresh=0x7f020049;
+        public static final int saved=0x7f02004a;
+        public static final int search=0x7f02004b;
+        public static final int select_album_play_all=0x7f02004c;
+        public static final int select_album_play_all_normal=0x7f02004d;
+        public static final int select_album_play_all_pressed=0x7f02004e;
+        public static final int select_artist_folder=0x7f02004f;
+        public static final int slider_knob=0x7f020050;
+        public static final int stat_notify_playing=0x7f020051;
+        public static final int unknown_album=0x7f020052;
+        public static final int unknown_album_large=0x7f020053;
+    }
+    public static final class id {
+        public static final int action_button_1=0x7f0c0002;
+        public static final int action_button_2=0x7f0c0001;
+        public static final int actionbar_home_icon=0x7f0c0000;
+        public static final int actionbar_title_text=0x7f0c0003;
+        public static final int album_appwidget=0x7f0c0007;
+        public static final int album_artist=0x7f0c0006;
+        public static final int album_coverart=0x7f0c0004;
+        public static final int album_title=0x7f0c0005;
+        public static final int artist=0x7f0c0009;
+        public static final int button_bar=0x7f0c000c;
+        public static final int button_bar_home=0x7f0c000d;
+        public static final int button_bar_music=0x7f0c000e;
+        public static final int button_bar_now_playing=0x7f0c0011;
+        public static final int button_bar_playlists=0x7f0c0010;
+        public static final int button_bar_search=0x7f0c000f;
+        public static final int control_next=0x7f0c000b;
+        public static final int control_play=0x7f0c000a;
+        public static final int download_album=0x7f0c0024;
+        public static final int download_album_art_image=0x7f0c001f;
+        public static final int download_album_art_layout=0x7f0c001e;
+        public static final int download_artist=0x7f0c0023;
+        public static final int download_button_bar_flipper=0x7f0c0020;
+        public static final int download_control_layout=0x7f0c0012;
+        public static final int download_duration=0x7f0c0022;
+        public static final int download_empty=0x7f0c0025;
+        public static final int download_list=0x7f0c0026;
+        public static final int download_next=0x7f0c0017;
+        public static final int download_pause=0x7f0c0014;
+        public static final int download_playlist_flipper=0x7f0c001d;
+        public static final int download_position=0x7f0c0021;
+        public static final int download_previous=0x7f0c0013;
+        public static final int download_progress_bar=0x7f0c0027;
+        public static final int download_repeat=0x7f0c001b;
+        public static final int download_shuffle=0x7f0c001a;
+        public static final int download_song_title=0x7f0c0019;
+        public static final int download_start=0x7f0c0016;
+        public static final int download_status=0x7f0c0018;
+        public static final int download_stop=0x7f0c0015;
+        public static final int download_toggle_list=0x7f0c001c;
+        public static final int help_back=0x7f0c0029;
+        public static final int help_buttons=0x7f0c0028;
+        public static final int help_close=0x7f0c002a;
+        public static final int help_contents=0x7f0c002b;
+        public static final int lyrics_artist=0x7f0c002c;
+        public static final int lyrics_text=0x7f0c002e;
+        public static final int lyrics_title=0x7f0c002d;
+        public static final int main_select_server_1=0x7f0c0032;
+        public static final int main_select_server_2=0x7f0c0033;
+        public static final int main_albums=0x7f0c0034;
+        public static final int main_albums_frequent=0x7f0c0037;
+        public static final int main_albums_highest=0x7f0c0038;
+        public static final int main_albums_newest=0x7f0c0035;
+        public static final int main_albums_random=0x7f0c0039;
+        public static final int main_albums_recent=0x7f0c0036;
+        public static final int main_dummy=0x7f0c0030;
+        public static final int main_list=0x7f0c002f;
+        public static final int main_select_server=0x7f0c0031;
+        public static final int menu_exit=0x7f0c005f;
+        public static final int menu_help=0x7f0c0061;
+        public static final int menu_lyrics=0x7f0c0067;
+        public static final int menu_play_all=0x7f0c006a;
+        public static final int menu_play_last=0x7f0c006f;
+        public static final int menu_play_next=0x7f0c006e;
+        public static final int menu_play_now=0x7f0c006d;
+        public static final int menu_queue_all=0x7f0c006b;
+        public static final int menu_queue_next=0x7f0c0069;
+        public static final int menu_remove=0x7f0c0068;
+        public static final int menu_remove_all=0x7f0c0064;
+        public static final int menu_save_all=0x7f0c006c;
+        public static final int menu_save_playlist=0x7f0c0063;
+        public static final int menu_screen_option=0x7f0c0065;
+        public static final int menu_settings=0x7f0c0060;
+        public static final int menu_show_album=0x7f0c0066;
+        public static final int menu_shuffle=0x7f0c0062;
+        public static final int play_video_contents=0x7f0c003a;
+        public static final int progress_message=0x7f0c003b;
+        public static final int save_playlist_name=0x7f0c003d;
+        public static final int save_playlist_root=0x7f0c003c;
+        public static final int search_albums=0x7f0c0041;
+        public static final int search_artists=0x7f0c0040;
+        public static final int search_list=0x7f0c003e;
+        public static final int search_more_albums=0x7f0c0044;
+        public static final int search_more_artists=0x7f0c0043;
+        public static final int search_more_songs=0x7f0c0045;
+        public static final int search_search=0x7f0c003f;
+        public static final int search_songs=0x7f0c0042;
+        public static final int select_album_cover_art=0x7f0c004e;
+        public static final int select_album_delete=0x7f0c004c;
+        public static final int select_album_empty=0x7f0c0046;
+        public static final int select_album_entries=0x7f0c0047;
+        public static final int select_album_more=0x7f0c004d;
+        public static final int select_album_play=0x7f0c0049;
+        public static final int select_album_play_all=0x7f0c0051;
+        public static final int select_album_queue=0x7f0c004a;
+        public static final int select_album_save=0x7f0c004b;
+        public static final int select_album_select=0x7f0c0048;
+        public static final int select_album_text1=0x7f0c004f;
+        public static final int select_album_text2=0x7f0c0050;
+        public static final int select_artist_folder=0x7f0c0053;
+        public static final int select_artist_folder_1=0x7f0c0054;
+        public static final int select_artist_folder_2=0x7f0c0055;
+        public static final int select_artist_list=0x7f0c0052;
+        public static final int select_playlist_empty=0x7f0c0056;
+        public static final int select_playlist_list=0x7f0c0057;
+        public static final int song_artist=0x7f0c005b;
+        public static final int song_check=0x7f0c0058;
+        public static final int song_duration=0x7f0c005c;
+        public static final int song_status=0x7f0c005a;
+        public static final int song_title=0x7f0c0059;
+        public static final int tab_progress=0x7f0c005d;
+        public static final int tab_progress_message=0x7f0c005e;
+        public static final int title=0x7f0c0008;
+    }
+    public static final class layout {
+        public static final int action_bar=0x7f030000;
+        public static final int album_list_item=0x7f030001;
+        public static final int appwidget=0x7f030002;
+        public static final int artist_list_item=0x7f030003;
+        public static final int button_bar=0x7f030004;
+        public static final int download=0x7f030005;
+        public static final int download_button_bar_flipper=0x7f030006;
+        public static final int download_playlist=0x7f030007;
+        public static final int download_slider=0x7f030008;
+        public static final int help=0x7f030009;
+        public static final int lyrics=0x7f03000a;
+        public static final int main=0x7f03000b;
+        public static final int main_buttons=0x7f03000c;
+        public static final int play_video=0x7f03000d;
+        public static final int playlist_list_item=0x7f03000e;
+        public static final int progress=0x7f03000f;
+        public static final int save_playlist=0x7f030010;
+        public static final int search=0x7f030011;
+        public static final int search_buttons=0x7f030012;
+        public static final int select_album=0x7f030013;
+        public static final int select_album_footer=0x7f030014;
+        public static final int select_album_header=0x7f030015;
+        public static final int select_artist=0x7f030016;
+        public static final int select_artist_header=0x7f030017;
+        public static final int select_playlist=0x7f030018;
+        public static final int song_list_item=0x7f030019;
+        public static final int tab_progress=0x7f03001a;
+    }
+    public static final class menu {
+        public static final int main=0x7f0b0000;
+        public static final int nowplaying=0x7f0b0001;
+        public static final int nowplaying_context=0x7f0b0002;
+        public static final int queue_context=0x7f0b0003;
+        public static final int select_album_context=0x7f0b0004;
+        public static final int select_artist_context=0x7f0b0005;
+        public static final int select_song_context=0x7f0b0006;
+    }
+    public static final class plurals {
+        public static final int select_album_donate_dialog_n_trial_days_left=0x7f090003;
+        public static final int select_album_n_songs=0x7f090000;
+        public static final int select_album_n_songs_added=0x7f090002;
+        public static final int select_album_n_songs_downloading=0x7f090001;
+    }
+    public static final class string {
+        public static final int background_task_loading=0x7f08009a;
+        public static final int background_task_network_error=0x7f08009c;
+        public static final int background_task_no_network=0x7f08009b;
+        public static final int background_task_not_found=0x7f08009d;
+        public static final int background_task_parse_error=0x7f08009e;
+        public static final int background_task_wait=0x7f080099;
+        public static final int common_appname=0x7f080000;
+        public static final int common_cancel=0x7f080003;
+        public static final int common_ok=0x7f080001;
+        public static final int common_save=0x7f080002;
+        public static final int download_empty=0x7f080041;
+        public static final int download_menu_lyrics=0x7f080046;
+        public static final int download_menu_remove=0x7f080047;
+        public static final int download_menu_remove_all=0x7f080048;
+        public static final int download_menu_save=0x7f08004b;
+        public static final int download_menu_screen_option=0x7f080049;
+        public static final int download_menu_show_album=0x7f080045;
+        public static final int download_menu_shuffle=0x7f08004a;
+        public static final int download_menu_shuffle_notification=0x7f08004c;
+        public static final int download_play_error=0x7f08004d;
+        public static final int download_playerstate_buffering=0x7f080043;
+        public static final int download_playerstate_downloading=0x7f080042;
+        public static final int download_playerstate_playing_shuffle=0x7f080044;
+        public static final int download_playlist_done=0x7f080052;
+        public static final int download_playlist_error=0x7f080053;
+        public static final int download_playlist_name=0x7f08004f;
+        public static final int download_playlist_saving=0x7f080051;
+        public static final int download_playlist_title=0x7f08004e;
+        public static final int download_playlist_unnamed=0x7f080050;
+        public static final int error_label=0x7f080057;
+        public static final int help_back=0x7f080016;
+        public static final int help_close=0x7f080017;
+        public static final int help_label=0x7f080014;
+        public static final int help_loading=0x7f080019;
+        public static final int help_title=0x7f080015;
+        public static final int help_url=0x7f080018;
+        public static final int lyrics_nomatch=0x7f080056;
+        public static final int main_albums_frequent=0x7f08000d;
+        public static final int main_albums_highest=0x7f08000e;
+        public static final int main_albums_newest=0x7f08000b;
+        public static final int main_albums_random=0x7f08000f;
+        public static final int main_albums_recent=0x7f08000c;
+        public static final int main_albums_title=0x7f08000a;
+        public static final int main_offline=0x7f080008;
+        public static final int main_select_server=0x7f080006;
+        public static final int main_settings=0x7f080009;
+        public static final int main_shuffle=0x7f080007;
+        public static final int main_welcome_text=0x7f080005;
+        public static final int main_welcome_title=0x7f080004;
+        public static final int menu_exit=0x7f080010;
+        public static final int menu_help=0x7f080012;
+        public static final int menu_settings=0x7f080011;
+        public static final int music_library_label=0x7f080025;
+        public static final int music_library_label_offline=0x7f080026;
+        /** <string name="settings.screen_lit_title">TODO: Keep screen on</string>
+<string name="settings.screen_lit_summary">TODO: Keeping the screen on when downloading may improve download speed</string>
+         */
+        public static final int music_service_retry=0x7f080098;
+        public static final int parser_artist_count=0x7f0800a5;
+        public static final int parser_not_authenticated=0x7f0800a4;
+        public static final int parser_reading=0x7f0800a0;
+        public static final int parser_reading_done=0x7f0800a1;
+        public static final int parser_upgrade_client=0x7f0800a2;
+        public static final int parser_upgrade_server=0x7f0800a3;
+        public static final int play_video_loading=0x7f08001a;
+        public static final int play_video_noplugin=0x7f08001b;
+        /** <string name="menu.exit">TODO: Exit</string>
+<string name="menu.settings">TODO: Settings</string>
+<string name="menu.help">TODO: Help</string>
+         */
+        public static final int playlist_label=0x7f080013;
+        public static final int progress_wait=0x7f080024;
+        public static final int queue_context_queue_next=0x7f08003f;
+        public static final int search_albums=0x7f080021;
+        public static final int search_artists=0x7f080020;
+        public static final int search_label=0x7f08001c;
+        public static final int search_more=0x7f080023;
+        public static final int search_no_match=0x7f08001f;
+        public static final int search_search=0x7f08001e;
+        public static final int search_songs=0x7f080022;
+        public static final int search_title=0x7f08001d;
+        public static final int select_album_delete=0x7f080031;
+        public static final int select_album_donate_dialog_0_trial_days_left=0x7f08003b;
+        public static final int select_album_donate_dialog_later=0x7f08003a;
+        public static final int select_album_donate_dialog_message=0x7f080038;
+        public static final int select_album_donate_dialog_now=0x7f080039;
+        public static final int select_album_empty=0x7f080027;
+        public static final int select_album_more=0x7f08002d;
+        public static final int select_album_n_selected=0x7f080029;
+        public static final int select_album_n_unselected=0x7f08002a;
+        public static final int select_album_no_network=0x7f080036;
+        public static final int select_album_no_sdcard=0x7f080035;
+        public static final int select_album_not_licensed=0x7f080037;
+        public static final int select_album_offline=0x7f080033;
+        /** <string name="select_album.n_selected">TODO: %d tracks selected.</string>
+<string name="select_album.n_unselected">TODO: %d tracks unselected.</string>
+         */
+        public static final int select_album_play=0x7f08002b;
+        public static final int select_album_play_all=0x7f08002e;
+        public static final int select_album_queue=0x7f08002c;
+        public static final int select_album_queue_all=0x7f08002f;
+        public static final int select_album_save=0x7f080032;
+        public static final int select_album_save_all=0x7f080030;
+        public static final int select_album_searching=0x7f080034;
+        public static final int select_album_select=0x7f080028;
+        public static final int select_artist_all_folders=0x7f0800a8;
+        public static final int select_artist_folder=0x7f0800a7;
+        public static final int select_artist_refresh=0x7f0800a6;
+        public static final int select_playlist_empty=0x7f080040;
+        public static final int select_song_play_last=0x7f08003e;
+        public static final int select_song_play_next=0x7f08003d;
+        public static final int select_song_play_now=0x7f08003c;
+        public static final int service_connecting=0x7f08009f;
+        public static final int settings_appearance_title=0x7f08006a;
+        public static final int settings_cache_size=0x7f080063;
+        public static final int settings_cache_size_100=0x7f080083;
+        public static final int settings_cache_size_1000=0x7f080086;
+        public static final int settings_cache_size_10000=0x7f080089;
+        public static final int settings_cache_size_200=0x7f080084;
+        public static final int settings_cache_size_2000=0x7f080087;
+        public static final int settings_cache_size_20000=0x7f08008a;
+        public static final int settings_cache_size_500=0x7f080085;
+        public static final int settings_cache_size_5000=0x7f080088;
+        public static final int settings_cache_size_unlimited=0x7f08008b;
+        public static final int settings_cache_title=0x7f080061;
+        public static final int settings_clear_search_history=0x7f08008c;
+        public static final int settings_connection_failure=0x7f080067;
+        public static final int settings_hide_media_summary=0x7f080092;
+        /** <string name="settings.scrobble_title">TODO: Scrobble to Last.fm</string>
+<string name="settings.scrobble_summary">TODO: Remember to set up your Last.fm user and password on the Subsonic server</string>
+         */
+        public static final int settings_hide_media_title=0x7f080091;
+        public static final int settings_hide_media_toast=0x7f080093;
+        public static final int settings_invalid_url=0x7f080068;
+        public static final int settings_invalid_username=0x7f080069;
+        public static final int settings_max_bitrate_112=0x7f080076;
+        public static final int settings_max_bitrate_128=0x7f080077;
+        public static final int settings_max_bitrate_160=0x7f080078;
+        public static final int settings_max_bitrate_192=0x7f080079;
+        public static final int settings_max_bitrate_256=0x7f08007a;
+        public static final int settings_max_bitrate_32=0x7f080072;
+        public static final int settings_max_bitrate_320=0x7f08007b;
+        public static final int settings_max_bitrate_64=0x7f080073;
+        public static final int settings_max_bitrate_80=0x7f080074;
+        public static final int settings_max_bitrate_96=0x7f080075;
+        public static final int settings_max_bitrate_mobile=0x7f080071;
+        public static final int settings_max_bitrate_unlimited=0x7f08007c;
+        public static final int settings_max_bitrate_wifi=0x7f080070;
+        public static final int settings_media_button_summary=0x7f080095;
+        public static final int settings_media_button_title=0x7f080094;
+        public static final int settings_network_title=0x7f08006f;
+        public static final int settings_other_title=0x7f08008e;
+        public static final int settings_preload=0x7f080062;
+        public static final int settings_preload_1=0x7f08007d;
+        public static final int settings_preload_10=0x7f080081;
+        public static final int settings_preload_2=0x7f08007e;
+        public static final int settings_preload_3=0x7f08007f;
+        public static final int settings_preload_5=0x7f080080;
+        public static final int settings_preload_unlimited=0x7f080082;
+        public static final int settings_screen_lit_summary=0x7f080097;
+        public static final int settings_screen_lit_title=0x7f080096;
+        public static final int settings_scrobble_summary=0x7f080090;
+        public static final int settings_scrobble_title=0x7f08008f;
+        public static final int settings_search_history_cleared=0x7f08008d;
+        public static final int settings_server_address=0x7f08005e;
+        public static final int settings_server_name=0x7f08005d;
+        public static final int settings_server_password=0x7f080060;
+        public static final int settings_server_unused1=0x7f08005b;
+        public static final int settings_server_unused2=0x7f08005c;
+        public static final int settings_server_username=0x7f08005f;
+        public static final int settings_servers_title=0x7f08005a;
+        public static final int settings_test_connection_title=0x7f080059;
+        public static final int settings_testing_connection=0x7f080064;
+        public static final int settings_testing_ok=0x7f080065;
+        public static final int settings_testing_unlicensed=0x7f080066;
+        public static final int settings_theme_dark=0x7f08006e;
+        public static final int settings_theme_light=0x7f08006d;
+        public static final int settings_theme_title=0x7f08006b;
+        public static final int settings_theme_wheat=0x7f08006c;
+        public static final int settings_title=0x7f080058;
+        public static final int song_details_all=0x7f080054;
+        public static final int song_details_kbps=0x7f080055;
+        public static final int util_bytes_format_byte=0x7f0800af;
+        public static final int util_bytes_format_gigabyte=0x7f0800ac;
+        public static final int util_bytes_format_kilobyte=0x7f0800ae;
+        public static final int util_bytes_format_megabyte=0x7f0800ad;
+        public static final int widget_initial_text=0x7f0800a9;
+        public static final int widget_sdcard_busy=0x7f0800aa;
+        public static final int widget_sdcard_missing=0x7f0800ab;
+    }
+    public static final class style {
+        public static final int Wheat=0x7f0a0000;
+        public static final int WheatListView=0x7f0a0001;
+    }
+    public static final class xml {
+        public static final int appwidget_info=0x7f050000;
+        public static final int searchable=0x7f050001;
+        public static final int settings=0x7f050002;
+    }
+}
diff -rupN ../../../android-release-2.8/proguard.cfg ./proguard.cfg
--- ../../../android-release-2.8/proguard.cfg	1969-12-31 16:00:00.000000000 -0800
+++ ./proguard.cfg	2011-06-15 15:26:10.000000000 -0700
@@ -0,0 +1,36 @@
+-optimizationpasses 5
+-dontusemixedcaseclassnames
+-dontskipnonpubliclibraryclasses
+-dontpreverify
+-verbose
+-optimizations !code/simplification/arithmetic,!field/*,!class/merging/*
+
+-keep public class * extends android.app.Activity
+-keep public class * extends android.app.Application
+-keep public class * extends android.app.Service
+-keep public class * extends android.content.BroadcastReceiver
+-keep public class * extends android.content.ContentProvider
+-keep public class * extends android.app.backup.BackupAgentHelper
+-keep public class * extends android.preference.Preference
+-keep public class com.android.vending.licensing.ILicensingService
+
+-keepclasseswithmembernames class * {
+    native <methods>;
+}
+
+-keepclasseswithmembernames class * {
+    public <init>(android.content.Context, android.util.AttributeSet);
+}
+
+-keepclasseswithmembernames class * {
+    public <init>(android.content.Context, android.util.AttributeSet, int);
+}
+
+-keepclassmembers enum * {
+    public static **[] values();
+    public static ** valueOf(java.lang.String);
+}
+
+-keep class * implements android.os.Parcelable {
+  public static final android.os.Parcelable$Creator *;
+}
Binary files ../../../android-release-2.8/res/.DS_Store and ./res/.DS_Store differ
Binary files ../../../android-release-2.8/res/drawable-hdpi/icon.png and ./res/drawable-hdpi/icon.png differ
Binary files ../../../android-release-2.8/res/drawable-ldpi/icon.png and ./res/drawable-ldpi/icon.png differ
Binary files ../../../android-release-2.8/res/drawable-mdpi/icon.png and ./res/drawable-mdpi/icon.png differ
diff -rupN ../../../android-release-2.8/res/menu/queue_context.xml ./res/menu/queue_context.xml
--- ../../../android-release-2.8/res/menu/queue_context.xml	1969-12-31 16:00:00.000000000 -0800
+++ ./res/menu/queue_context.xml	2011-06-17 08:34:06.000000000 -0700
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:a="http://schemas.android.com/apk/res/android">
+    <item 
+        a:id="@+id/menu_queue_next"
+        a:title="@string/queue_context.queue_next"
+        />        
+</menu>
\ No newline at end of file
diff -rupN ../../../android-release-2.8/res/menu/select_song_context.xml ./res/menu/select_song_context.xml
--- ../../../android-release-2.8/res/menu/select_song_context.xml	1969-12-31 16:00:00.000000000 -0800
+++ ./res/menu/select_song_context.xml	2011-06-16 14:51:54.000000000 -0700
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:a="http://schemas.android.com/apk/res/android">
+
+    <item 
+        a:id="@+id/menu_play_now"
+        a:title="@string/select_song.play_now"
+        />
+
+    <item 
+        a:id="@+id/menu_play_next"
+        a:title="@string/select_song.play_next"
+        />
+
+    <item 
+        a:id="@+id/menu_play_last"
+        a:title="@string/select_song.play_last"
+        />
+        
+</menu>
diff -rupN ../../../android-release-2.8/res/values/strings.xml ./res/values/strings.xml
--- ../../../android-release-2.8/res/values/strings.xml	2011-06-17 10:30:36.000000000 -0700
+++ ./res/values/strings.xml	2011-06-17 08:33:58.000000000 -0700
@@ -71,6 +71,10 @@
     <string name="select_album.donate_dialog_now">Now</string>
     <string name="select_album.donate_dialog_later">Later</string>
     <string name="select_album.donate_dialog_0_trial_days_left">Trial period is over</string>
+    <string name="select_song.play_now">Play song now</string>
+    <string name="select_song.play_next">Play song next</string>
+    <string name="select_song.play_last">Play song last</string>    
+    <string name="queue_context.queue_next">Queue songs next</string>    
 
     <string name="select_playlist.empty">No saved playlists on server</string>
 
diff -rupN ../../../android-release-2.8/src/net/sourceforge/subsonic/androidapp/activity/SearchActivity.java ./src/net/sourceforge/subsonic/androidapp/activity/SearchActivity.java
--- ../../../android-release-2.8/src/net/sourceforge/subsonic/androidapp/activity/SearchActivity.java	2011-06-17 10:26:41.000000000 -0700
+++ ./src/net/sourceforge/subsonic/androidapp/activity/SearchActivity.java	2011-06-17 09:30:44.000000000 -0700
@@ -354,7 +354,7 @@ public class SearchActivity extends Subs
             if (!append) {
                 downloadService.clear();
             }
-            downloadService.download(Arrays.asList(song), save, false);
+            downloadService.download(Arrays.asList(song), save, false, false);
             if (autoplay) {
                 downloadService.play(downloadService.size() - 1);
             }
diff -rupN ../../../android-release-2.8/src/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity.java ./src/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity.java
--- ../../../android-release-2.8/src/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity.java	2011-06-17 10:26:41.000000000 -0700
+++ ./src/net/sourceforge/subsonic/androidapp/activity/SelectAlbumActivity.java	2011-06-17 10:22:03.000000000 -0700
@@ -114,21 +114,21 @@ public class SelectAlbumActivity extends
         playButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                download(false, false, true);
+                download(false, false, true, false);
                 selectAll(false, false);
             }
         });
         queueButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                download(true, false, false);
+                download(true, false, false, false);
                 selectAll(false, false);
             }
         });
         saveButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                download(true, true, false);
+                download(true, true, false, false);
                 selectAll(false, false);
             }
         });
@@ -141,6 +141,7 @@ public class SelectAlbumActivity extends
         });
 
         registerForContextMenu(entryList);
+        registerForContextMenu(queueButton);
 
         enableButtons();
 
@@ -197,7 +198,7 @@ public class SelectAlbumActivity extends
             downloadRecursively(id, false, false, true);
         } else {
             selectAll(true, false);
-            download(false, false, true);
+            download(false, false, true, false);
             selectAll(false, false);
         }
     }
@@ -212,32 +213,58 @@ public class SelectAlbumActivity extends
     @Override
     public void onCreateContextMenu(ContextMenu menu, View view, ContextMenu.ContextMenuInfo menuInfo) {
         super.onCreateContextMenu(menu, view, menuInfo);
-
-        AdapterView.AdapterContextMenuInfo info = (AdapterView.AdapterContextMenuInfo) menuInfo;
-        MusicDirectory.Entry entry = (MusicDirectory.Entry) entryList.getItemAtPosition(info.position);
-        if (entry.isDirectory()) {
+    	if (view == queueButton) {
             MenuInflater inflater = getMenuInflater();
-            inflater.inflate(R.menu.select_album_context, menu);
-        }
+            inflater.inflate(R.menu.queue_context, menu);
+    	} else {
+            AdapterView.AdapterContextMenuInfo info =
+                (AdapterView.AdapterContextMenuInfo) menuInfo;
+        	
+            MusicDirectory.Entry entry = (MusicDirectory.Entry) entryList.getItemAtPosition(info.position);
+
+            if (entry.isDirectory()) {
+                MenuInflater inflater = getMenuInflater();
+                inflater.inflate(R.menu.select_album_context, menu);
+            } else {
+                MenuInflater inflater = getMenuInflater();
+                inflater.inflate(R.menu.select_song_context, menu);
+            }    		
+    	}        
     }
 
     @Override
     public boolean onContextItemSelected(MenuItem menuItem) {
-        AdapterView.AdapterContextMenuInfo info = (AdapterView.AdapterContextMenuInfo) menuItem.getMenuInfo();
-        MusicDirectory.Entry entry = (MusicDirectory.Entry) entryList.getItemAtPosition(info.position);
-        switch (menuItem.getItemId()) {
-            case R.id.menu_play_all:
-                downloadRecursively(entry.getId(), false, false, true);
-                break;
-            case R.id.menu_queue_all:
-                downloadRecursively(entry.getId(), false, true, false);
-                break;
-            case R.id.menu_save_all:
-                downloadRecursively(entry.getId(), true, true, false);
-                break;
-            default:
-                return super.onContextItemSelected(menuItem);
-        }
+    	if (menuItem.getItemId() == R.id.menu_queue_next) {
+            download(true, false, false, true);
+            selectAll(false, false);    		
+    	} else {    		
+        	AdapterView.AdapterContextMenuInfo info = (AdapterView.AdapterContextMenuInfo) menuItem.getMenuInfo();
+            MusicDirectory.Entry entry = (MusicDirectory.Entry) entryList.getItemAtPosition(info.position);
+            List<MusicDirectory.Entry> songs = new ArrayList<MusicDirectory.Entry>(10);
+            songs.add((MusicDirectory.Entry) entryList.getItemAtPosition(info.position));
+            switch (menuItem.getItemId()) {
+                case R.id.menu_play_all:
+                    downloadRecursively(entry.getId(), false, false, true);
+                    break;
+                case R.id.menu_queue_all:
+                    downloadRecursively(entry.getId(), false, true, false);
+                    break;
+                case R.id.menu_save_all:
+                    downloadRecursively(entry.getId(), true, true, false);
+                    break;
+                case R.id.menu_play_now:
+                	getDownloadService().download(songs, false, true, true);
+                    break;
+                case R.id.menu_play_next:
+                	getDownloadService().download(songs, false, false, true);
+                    break;
+                case R.id.menu_play_last:
+                    getDownloadService().download(songs, false, false, false);
+                    break;
+                default:
+                    return super.onContextItemSelected(menuItem);
+            }
+    	}
         return true;
     }
 
@@ -379,7 +406,7 @@ public class SelectAlbumActivity extends
         return songs;
     }
 
-    private void download(final boolean append, final boolean save, final boolean autoplay) {
+    private void download(final boolean append, final boolean save, final boolean autoplay, final boolean play_next) {
         if (getDownloadService() == null) {
             return;
         }
@@ -393,7 +420,7 @@ public class SelectAlbumActivity extends
                 }
 
                 warnIfNetworkOrStorageUnavailable();
-                getDownloadService().download(songs, save, autoplay);
+                getDownloadService().download(songs, save, autoplay, play_next);
                 getDownloadService().setSuggestedPlaylistName(getIntent().getStringExtra(Constants.INTENT_EXTRA_NAME_PLAYLIST_NAME));
                 if (autoplay) {
                     Util.startActivityWithoutTransition(SelectAlbumActivity.this, DownloadActivity.class);
diff -rupN ../../../android-release-2.8/src/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity.java ./src/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity.java
--- ../../../android-release-2.8/src/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity.java	2011-06-17 10:26:41.000000000 -0700
+++ ./src/net/sourceforge/subsonic/androidapp/activity/SubsonicTabActivity.java	2011-06-17 09:30:11.000000000 -0700
@@ -287,7 +287,7 @@ public class SubsonicTabActivity extends
                         downloadService.clear();
                     }
                     warnIfNetworkOrStorageUnavailable();
-                    downloadService.download(songs, save, autoplay);
+                    downloadService.download(songs, save, autoplay, false);
                     Util.startActivityWithoutTransition(SubsonicTabActivity.this, DownloadActivity.class);
                 }
             }
diff -rupN ../../../android-release-2.8/src/net/sourceforge/subsonic/androidapp/service/DownloadService.java ./src/net/sourceforge/subsonic/androidapp/service/DownloadService.java
--- ../../../android-release-2.8/src/net/sourceforge/subsonic/androidapp/service/DownloadService.java	2011-06-17 10:26:42.000000000 -0700
+++ ./src/net/sourceforge/subsonic/androidapp/service/DownloadService.java	2011-06-17 09:29:09.000000000 -0700
@@ -30,7 +30,7 @@ import java.util.List;
  */
 public interface DownloadService {
 
-    void download(List<MusicDirectory.Entry> songs, boolean save, boolean autoplay);
+    void download(List<MusicDirectory.Entry> songs, boolean save, boolean autoplay, boolean play_next);
 
     void setShufflePlayEnabled(boolean enabled);
 
diff -rupN ../../../android-release-2.8/src/net/sourceforge/subsonic/androidapp/service/DownloadServiceImpl.java ./src/net/sourceforge/subsonic/androidapp/service/DownloadServiceImpl.java
--- ../../../android-release-2.8/src/net/sourceforge/subsonic/androidapp/service/DownloadServiceImpl.java	2011-06-17 10:26:42.000000000 -0700
+++ ./src/net/sourceforge/subsonic/androidapp/service/DownloadServiceImpl.java	2011-06-17 10:25:53.000000000 -0700
@@ -129,19 +129,29 @@ public class DownloadServiceImpl extends
     }
 
     @Override
-    public synchronized void download(List<MusicDirectory.Entry> songs, boolean save, boolean autoplay) {
+    public synchronized void download(List<MusicDirectory.Entry> songs, boolean save, boolean autoplay, boolean play_next) {
         shufflePlay = false;
+        int offset = 1;
 
         if (songs.isEmpty()) {
             return;
         }
-
-        for (MusicDirectory.Entry song : songs) {
-            DownloadFile downloadFile = new DownloadFile(this, song, save);
-            downloadList.add(downloadFile);
+        if (play_next) {
+        	if ((autoplay) && getCurrentPlayingIndex() >= 0) offset = 0;
+            for (MusicDirectory.Entry song : songs) {
+            	DownloadFile downloadFile = new DownloadFile(this, song, save);
+                downloadList.add(getCurrentPlayingIndex() + offset, downloadFile);
+                offset++;
+            }
+            revision++;
+        } else {
+            for (MusicDirectory.Entry song : songs) {
+                DownloadFile downloadFile = new DownloadFile(this, song, save);
+                downloadList.add(downloadFile);
+            }
+            revision++;        	
         }
-        revision++;
-
+        
         if (autoplay) {
             play(0);
         } else {
@@ -154,7 +164,7 @@ public class DownloadServiceImpl extends
     }
 
     public void restore(List<MusicDirectory.Entry> songs, int currentPlayingIndex, int currentPlayingPosition) {
-        download(songs, false, false);
+        download(songs, false, false, false);
         if (currentPlayingIndex != -1) {
             play(currentPlayingIndex, false);
             if (currentPlaying.isCompleteFileAvailable()) {
